src/python_pybind11/test/DiffDriveOdometry_TEST.py

```diff
@@ -71,7 +72,7 @@ def test_diff_drive_odometry(self):
         self.assertAlmostEqual(0.0, odom.angular_velocity().radian(), delta=1e-3)
 
         # Initialize again, and odom values should be reset.
-        startTime = datetime.datetime.now()
+        startTime = datetime.timedelta(time.monotonic())
         odom.init(datetime.timedelta())
```

Comment:
```suggestion
        startTime = datetime.timedelta(time.monotonic())
        odom.init(startTime)
```


then update
`odom.update(..., ..., time1)`